### PR TITLE
Example: zip

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: Tasks/zip.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    }
+  else
+    {
+      (() => (b[j++]=CELL))();
+    }
+  }
+return b
+}
+
+const result = zip(['Roma', 'Kiev', 'Beijing'], [100, 200, 300]);
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:20]: console.log(zip(['Roma', 'Kiev', 'Beijing', 'Oslo' ], [100, 200, 300, 199]));

File: Tasks/zip.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+let i = 0; j = 0
+  for (x of b) {
+    CELL = [a[i++],x];
+    if (i<j)
+    {
+      delete a[i++]
+    }
+  else
+    {
+      (() => (b[j++]=CELL))();
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:14]: arrOutput[j++]=CELL

File: Tasks/zip.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Find first word with two identical letters
+
+const zip = function ( a = [], b = [])
+{
+let i = 0; j = 0
+  for (x of b) {
+    CELL = [a[i++],x];
+    if (i<j)
+    {
+      delete a[i++]
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:10]: delete arrInput[i++]

File: Tasks/zip.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// Find first word with two identical letters
+
+const zip = function ( a = [], b = [])
+{
+let i = 0; j = 0
+  for (x of b) {
+    CELL = [a[i++],x];
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:7]: CELL = [arrInput[i++], x]

File: Tasks/zip.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+// Find first word with two identical letters
+
+const zip = function ( a = [], b = [])
+{
+let i = 0; j = 0
+  for (x of b) {
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:6]: for (x of arrOutput)

File: Tasks/zip.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+// Find first word with two identical letters
+
+const zip = function ( a = [], b = [])
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:3]: const zip = function (arrInput = [], arrOutput= [])

File: Tasks/zip.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  else
+    {
+      (() => (b[j++]=CELL))();
+    }
+  }
+return b
+}
+
+const result = zip(['Roma', 'Kiev', 'Beijing'], [100, 200, 300]);
+console.log(result);
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:21]: removed it

File: Tasks/zip.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    if (i<j)
+    {
+      delete a[i++]
+    }
+  else
+    {
+      (() => (b[j++]=CELL))();
+    }
+  }
+return b
```
Comment (Zakhar-Khmurych) [inline Tasks/zip.js:17]: return arrOutput;

**Review by Zakhar-Khmurych — COMMENTED** [review]
Minor fixes

**Comment by Zakhar-Khmurych** [comment]
```
js
const zip = function (arrInput = [], arrOutput= [])
{
    let i = 0, j = 0;
        for (x of arrOutput)
        {
            CELL = [arrInput[i++], x]
            if (i<j)
            {
                delete arrInput[i++]
            }
            else
            {
                arrOutput[j++]=CELL
            }
        }
    return arrOutput;
}

console.log(zip(['Roma', 'Kiev', 'Beijing', 'Oslo' ], [100, 200, 300, 199]));
```

**Comment by Zakhar-Khmurych** [comment]
```
#python 

def array_exchange(arr_1 = [], arr_2 = []):

    arr_result = []
    arr_temp = []
    check_stop = False
    i = 0

    arr1_len = len(arr_1)
    arr2_len = len(arr_2)

    while check_stop == False:
        if (arr1_len <= i) and (arr2_len<=i):
            check_stop = True
            break

        arr_temp = [str(arr_1[i]), str(arr_2[i])]
        i += 1
        arr_result.append(arr_temp)
    return arr_result

output = array_exchange(['Roma', 'Kiev', 'Beijing', 'Stockholm'], [100, 200, 300, 500])

print(output)
```

**Comment by tshemsedinov** [comment]
- Use prettier
-

**Pull request closed without merging**